p2panda-rs/src/operation/traits.rs

```diff
@@ -28,7 +28,7 @@ pub trait Actionable {
 /// against a schema.
 pub trait Schematic {
     /// Returns the schema id.
-    fn schema_id(&self) -> &SchemaId;
+    fn schema_id(&self) -> SchemaId;
```

Comment:
I'd still borrow here. Most validation methods will never need to own any objects and it's more flexible.